velox/exec/SortBuffer.cpp

```diff
@@ -90,9 +90,8 @@ void SortBuffer::addInput(const VectorPtr& input) {
   for (const auto& columnProjection : columnMap_) {
     DecodedVector decoded(
         *inputRow->childAt(columnProjection.outputChannel), allRows);
-    for (int i = 0; i < input->size(); ++i) {
-      data_->store(decoded, i, rows[i], columnProjection.inputChannel);
-    }
+    data_->storeVector(
```

Comment:
This is a nice refactoring. It seems unrelated to nulls optimization though. Would it be possible to extract this into a separate PR to help reduce the size of this PR and allow for easier faster review?

Reply:
This new function is based on some changes in nulls optimization. And now the change is smaller than before, please let me know if i still need to move it to antoher PR, thanks.

Reply:
Yes, let's extract this change into a separate PR. This change is logically separate and much safer than the change for tracking nulls. In case null tracking change causes a regression and needs to be reverted, the revert will be smaller.

Reply:
Please help to take a look, thanks.
https://github.com/facebookincubator/velox/pull/10812